include/mbedtls/ccm.h

```diff
@@ -85,17 +84,17 @@ mbedtls_ccm_context;
  *                  to make references valid, and prepare the context
  *                  for mbedtls_ccm_setkey() or mbedtls_ccm_free().
  *
- * \param ctx       The CCM context to initialize.
+ * \param ctx       The CCM context to initialize. Must not be NULL.
  */
 void mbedtls_ccm_init( mbedtls_ccm_context *ctx );
 
 /**
  * \brief           This function initializes the CCM context set in the
  *                  \p ctx parameter and sets the encryption key.
  *
- * \param ctx       The CCM context to initialize.
+ * \param ctx       The CCM context to initialize. Must not be NULL.
  * \param cipher    The 128-bit block cipher to use.
- * \param key       The encryption key.
+ * \param key       The encryption key. Must not be NULL.
```

Comment:
I think it's better to do as Hanno does in #2272 and mention "this must be a readable buffer of size ..."

Reply:
And, if you agree with that approach, require that contexts should be initialized, as opposed to just not NULL.